controllers/controllers.js

```diff
@@ -11,6 +11,16 @@ exports.getApi = (req, res, next) =>{
 
 exports.getTopics = (req, res, next) =>{
     return fetchTopics().then((queryResponse)=>{
-        res.status(200).send({topics: queryResponse.rows})
+        res.status(200).send({topics: queryResponse})
```

Comment:
There is an easy way to refactor this so that you only have to send `{ topics }`